test/Conversion/amd/mfma-shortcut.mlir

```diff
@@ -1,4 +1,5 @@
 // RUN: triton-opt %s --tritongpu-reduce-data-duplication --allocate-shared-memory --convert-triton-amdgpu-to-llvm=arch="gfx942" -split-input-file | FileCheck %s
+// RUN: triton-opt %s --tritongpu-reduce-data-duplication --allocate-shared-memory --convert-triton-amdgpu-to-llvm=arch="gfx950" -split-input-file | FileCheck %s --check-prefix=GFX950
```

Comment:
Can you change the existing tests to use `--check-prefix=GFX942` to be symmetric?